src/BasketBundle/Block/BasketBlockService.php

```diff
@@ -24,7 +24,7 @@
 /**
  * @author Xavier Coureau <xcoureau@ekino.com>
  */
-class BasketBlockService extends AbstractAdminBlockService
+class BasketBlockService extends AbstractBlockService
```

Comment:
BasketBlockService does not imprement AdminBlockServiceInterface so it is BC-break. Anyway you should extends EditableBlockService.

Reply:
i just leave as i was. see next time